scripts/ZipToGraphics/zipToGraphics.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+'''
+zipToGraphics
+----------------------------------
+Given a .zip file with a shapefile it validates and then returns the feature as a graphic
+'''
+from zipfile import ZipFile
+from os.path import join
+import arcpy
+
+
+required_files = ['shp', 'dbf', 'prj', 'shx']
+wgs = arcpy.SpatialReference(3857)
+
+multiple_warning = 'warning:Multiple features were found in the uploaded shapefile but only the first was returned.'
+categories = {
+    'Terrestrial Treatment Area': ['Polygon'],
+    'Aquatic/Riparian Treatment Area': ['Polygon'],
+    'Affected Area': ['Polygon'],
+    'Easement/Acquisition': ['Polygon'],
+    'Guzzler': ['Point', 'Multipoint'],
```

Comment:
This doesn't control what the return geometry type is. It's just a check on the input geometry type.